src/storage/engine/mod.rs

```diff
@@ -124,6 +124,9 @@ pub trait Engine: Send + Debug {
 pub trait Snapshot: Send + Debug {
     fn get(&self, key: &Key) -> Result<Option<Value>>;
     fn get_cf(&self, cf: CfName, key: &Key) -> Result<Option<Value>>;
+    fn batch_get(&self, keys: &[Key]) -> Result<Vec<Result<Option<Value>>>> {
```

Comment:
Should also add `batch_get_cf`.

Reply:
yes, I think we can use get outside directly.

Reply:
It's changed to use get directly. And please check the comments I left about the empty lines. I'm not sure exactly what kind of style is better.